components/MessageBox/Index.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <div
     class="w-[1038px] h-40 rounded-md flex text-white"
```

Comment:
`w-[1038px]` is static style and this is not what we expect. It is better to change it to `w-full`